framework/Makefile

```diff
@@ -46,6 +46,11 @@ ifeq ($(CONFIG_WIFI_MANAGER), y)
 include src$(DELIM)wifi_manager$(DELIM)Make.defs
 endif
 
+ifeq ($(CONFIG_ST_THINGS), y)
+CFLAGS += -I$(TOPDIR)/../framework/include/st_things
```

Comment:
The framework/include folder is already included. Do you need to include this?
When user uses st_things header, they should use like below.
```
#include <st_things/xx.h>
```
But in st_things framework, it will be used like below.
```
#include <xx.h>
```

User who refer framework codes can be confused.
I left this comment at Junsik's PR and he modified. Do you want to rollback?

Reply:
modified